prow/plugins/lifecycle/reopen.go

```diff
@@ -49,21 +50,27 @@ func handleReopen(gc githubClient, log *logrus.Entry, e *github.GenericCommentEv
 	number := e.Number
 	commentAuthor := e.User.Login
 
-	// Allow assignees and authors to re-open issues.
-	isAssignee := false
-	for _, assignee := range e.Assignees {
-		if commentAuthor == assignee.Login {
-			isAssignee = true
-			break
-		}
+	isAuthor := e.IssueAuthor.Login == commentAuthor
+	isCollaborator, err := gc.IsCollaborator(org, repo, commentAuthor)
+	if err != nil {
+		log.WithError(err).Errorf("Failed IsCollaborator(%s, %s, %s)", org, repo, commentAuthor)
 	}
-	if e.IssueAuthor.Login != commentAuthor && !isAssignee {
-		resp := "you can't re-open an issue/PR unless you authored it or you are assigned to it."
-		log.Infof("Commenting \"%s\".", resp)
-		return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
+
+	// Only authors and collaborators are allowed to reopen issues or PRs.
+	if !isAuthor && !isCollaborator {
+		response := fmt.Sprintf("You can't reopen an issue/PR unless you authored it or you are a collaborator.")
```

Comment:
should we link to any guide here or at least a reference to what a "collaborator" is? it might not be clear what all counts for that

Reply:
I don't think it's needed. We don't have anything in place for LGTM which uses collaborator terminology. 🤷🏻‍♂️

Reply:
I haven't added a link for now. Let me know if I should!